services/web/pkg/config/options.go

```diff
@@ -0,0 +1,65 @@
+	DisableFeedbackLink      bool            `json:"disableFeedbackLink" yaml:"disableFeedbackLink" env:"WEB_OPTION_DISABLE_FEEDBACK_LINK" desc:"Set this option to 'true' to disable the feedback link in the topbar. Keeping it enabled (value 'false' or absence of the option) allows ownCloud to get feedback from your user base through a dedicated survey website."`
+	FeedbackLink             FeedbackLink    `json:"feedbackLink" yaml:"feedbackLink"`
+	SharingRecipientsPerPage int             `json:"sharingRecipientsPerPage" yaml:"sharingRecipientsPerPage" env:"WEB_OPTION_SHARING_RECIPIENTS_PER_PAGE" desc:"Sets the amount of users shown as recipients in the dropdown when sharing resources. Default amount is 200."`
+	Sidebar                  Sidebar         `json:"sidebar" yaml:"sidebar"`
+	RunningOnEOS             bool            `json:"runningOnEos" yaml:"runningOnEos" env:"WEB_OPTION_RUNNING_ON_EOS" desc:"Set this option to 'true' if running on an [EOS storage backend](https://eos-web.web.cern.ch/eos-web/) to enable its specific features. Defaults to 'false'."`
+	CernFeatures             bool            `json:"cernFeatures" yaml:"cernFeatures" env:"WEB_OPTION_CERN_FEATURES" desc:"Enabling this will activate CERN-specific features. Defaults to 'false'."`
+	HoverableQuickActions    bool            `json:"hoverableQuickActions" yaml:"hoverableQuickActions" env:"WEB_OPTION_HOVERABLE_QUICK_ACTIONS" desc:"Set this option to 'true' to hide the quick actions (buttons appearing on file rows), and only show them when the user hovers the row with his mouse. Defaults to 'false'."`
+	Routing                  Routing         `json:"routing" yaml:"routing"`
+	Upload                   Upload          `json:"upload" yaml:"upload"`
+	Editor                   Editor          `json:"editor" yaml:"editor"`
+	ContextHelpersReadMore   bool            `json:"contextHelpersReadMore" yaml:"contextHelpersReadMore" env:"WEB_OPTION_CONTEXTHELPERS_READ_MORE" desc:"Specifies whether the 'Read more' link should be displayed or not."`
+	LogoutURL                string          `json:"logoutURL" yaml:"logoutURL" env:"WEB_OPTION_LOGOUT_URL" desc:"The logout url"`
+}
+
+// AccountEditLink are the AccountEditLink options
+type AccountEditLink struct {
+	Href string `json:"href" yaml:"href" env:"WEB_OPTION_ACCOUNT_EDIT_LINK_HREF" desc:"Set a different target URL for the edit link. Make sure to prepend it with 'http(s)://'."`
+}
+
+// FeedbackLink are the feedback link options
+type FeedbackLink struct {
+	Href        string `json:"href" yaml:"href" env:"WEB_OPTION_FEEDBACKLINK_HREF" desc:"Set a different target URL for the feedback link. Make sure to prepend it with 'http(s)://'. Defaults to 'https://owncloud.com/web-design-feedback'.`
+	AriaLabel   string `json:"ariaLabel" yaml:"ariaLabel" env:"WEB_OPTION_ARIALABEL" desc:"Since the link only has an icon, you can set an e.g. screen reader accessible label. Defaults to 'ownCloud feedback survey'.`
```

Comment:
```suggestion
	AriaLabel   string `json:"ariaLabel" yaml:"ariaLabel" env:"WEB_OPTION_ARIALABEL" desc:"Since the feedback link only has an icon, a screen reader accessible label can be set. The text defaults to 'ownCloud feedback survey'.`
```